app/views/claim_mailer/approved.text.erb

```diff
@@ -0,0 +1,30 @@
+Dear <%= @display_name %>,
+
+We are pleased to say that your claim to get your student loan payments has been approved.
+
+We will now process your claim so we can calculate the amount of tax owed for this repayment, and then pay it for you. This will not affect the amount of money you will receive.
+
+To do this we will share your claim data with our payroll provider and HMRC. This will include:
+
+* name
+* email
+* address
+* date of birth
+* gender as recorded on your digital identity
```

Comment:
I'm not sure this will work well for trans people. If I got this email weeks after I submitted the claim, I don't know if I'd know what we're referrring to by "digital identity". Combining that with the line following this list would make me unsure about what data is being submitted. We know if the user told us their gender or if it came from Verify. Can we alter this text based on that information to be more explicit about what the source of the information we're sharing is? I think this applies for other fields (like name).

Reply:
@erbridge shall we raise this with the team as it's content, I wouldn't be comfatable changing this without their input?

Reply:
Yeah, sorry, I'm raising it here, but it definitely makes sense to talk to the team.